Source/Editor/SceneGraph/Actors/SplineNode.cs

```diff
@@ -398,6 +408,43 @@ internal static void OnSplineEdited(Spline spline)
             }
         }
 
+        internal static float NodeSizeByDistance(Vector3 nodePosition, float nodeSize)
```

Comment:
Same thing here, you need to replace all new mentions of `float` with `Real` instead, this way you will fix CI/CD compilation issue that you get from `LargeWorlds` compilation mode.

Reply:
@Withaust thanks